docs/tutorial-module/run-n-test.rst

```diff
@@ -17,16 +17,23 @@ Before that, we need to **stop** any running Invenio instance.
 
 Install the module
 ------------------
-Install the module is very easy, you just need to go to the root directory of
-the module and run the command:
+First, create a virtualenv for the module:
 
 .. code-block:: bash
 
-    pip install -e .[all]
+    $ mkvirtualenv my-module-venv
+
+Installing the module is very easy, you just need to go to the root directory
```

Comment:
Maybe a shorter version: `Installing the module is very easy, you just need to go to its root directory and run:` what do you think?

Reply:
Short and sweet, I like it.